docs/sessions.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: default
+---
+
+State and Sessions in Suave
+===========================
+
+While stateless web applications are all the rage, there are still times where state needs to be preserved, either across requests in a session, or simply among `WebPart`s as the request/response pipeline is composed.  In these cases, Suave has you covered!  We'll look at three aspects of session cookies first, then how we can construct state among `WebPart`s.
```

Comment:
How about

While web applications are technically stateless, there are still times where we need to start from a known state.  So, let's jump in and look at aspects of session cookies first, then how we can manage state among `WebPart`s.